frontend/badge-examples.js

```diff
@@ -21,9 +22,9 @@ const Badge = ({ title, previewUri, exampleUri, documentation, keywords, baseUri
     'data-keywords': keywords ? keywords.join(' ') : undefined,
   };
   const previewImage = previewUri
-    ? (<img src={resolvePreviewUri(previewUri, baseUri, isProductionBuild)} alt="" />)
+    ? (<img src={resolveUri(previewUri, baseUri, isProductionBuild)} alt="" />)
     : '\u00a0'; // non-breaking space
-  const resolvedExampleUri = url.resolve(baseUri, exampleUri || previewUri);
+  const resolvedExampleUri = resolveUri(exampleUri || previewUri, baseUri || 'https://img.shields.io/', false);
```

Comment:
Would it be more readable with an explicit boolean, like `resolveUri(…, { longCache: false })`?

Reply:
Sure, I like that idea.